examples/telegraf/disk/disk_alert_batch.tick

```diff
@@ -0,0 +1,38 @@
+// disk_alert_batch
+
+// metric: used_percent
+// available_fields: "free","inodes_free","inodes_total","inodes_used","total","used"
+
+// TELEGRAF CONFIGURATION
+// [[inputs.disk]]
+//   ignore_fs = ["tmpfs", "devtmpfs"]
+
+// DEFINE: kapacitor define disk_alert_batch -type batch -tick disk/disk_alert_batch.tick -dbrp telegraf.autogen
+// ENABLE: kapacitor enable disk_alert_batch
+
+// Parameters
+var info = 75 
+var warn = 85
+var crit = 92
+var period = 10s
+var every = 10s
+
+// Dataframe
+var data = batch
+  |query('''SELECT used_percent AS stat FROM "telegraf"."autogen"."disk" WHERE path = '/' ''')
```

Comment:
Should we also apply the `mean` function here? So that it is consistent with the stream version.